examples/unconditional_image_generation/train_unconditional.py

```diff
@@ -171,13 +171,26 @@ def parse_args():
     parser.add_argument(
         "--hub_private_repo", action="store_true", help="Whether or not to create a private repository."
     )
+    parser.add_argument(
+        "--logger",
+        type=str,
+        default="tensorboard",
+        choices=["tensorboard", "wandb"],
+        nargs="+",
```

Comment:
Could we maybe allow just one of the two? :-)

Reply:
To clarify: do you mean to reduce the number of args to allow only one logger? I can do so (look at the previous comment to see why we did not initially)

Reply:
I see thanks for the explanation! 
Sorry I mean maybe it be nicer to allow the user to only use **one** not multiple of the **choices**

Reply:
Yeah, good point, I agree